authz/policy-enforcer/src/main/java/org/keycloak/adapters/authorization/PathConfigMatcher.java

```diff
@@ -65,6 +65,20 @@ public class PathConfigMatcher extends PathMatcher<PathConfig> {
         }
     }
 
+    public boolean isEnforcementDisabled(String targetUri) {
+        PathConfig pathConfig = pathCache.get(targetUri);
+
+        if (pathConfig == null && !pathCache.containsKey(targetUri)) {
```

Comment:
```suggestion
        if (pathConfig == null) {
```

Reply:
`pathCache.containsKey(targetUri)` can be true even when `pathConfig == null` because even cases when path config wasn't found (`matches` does same check, `org.keycloak.adapters.authorization.PathCache#put`). Just by reading code, I guess this situation may happen when even any path config isn't matched (that's what code allows, not sure if application logic too).